### DISCUSSION TITLE
can I use web3auth for btc, doge, ltc and bch?

### DISCUSSION THREAD

**Discussion by hodler-smart** [Web3Auth/Web3Auth#1226] [Help]
I want to make wallet with web3auth for btc, doge, ltc and bch.
The Web3Auth Web SDK supports all blockchains following the secp256k1 & ed25519 curves. So I think it's possible to use web3auth for btc, doge ltc and bch.
Please  help me how to do it.

**Comment by arch1995** [comment]
Hi @alphadev102 

Web3auth is chain agnostic in nature.

Please follow any of the references - 
https://web3auth.io/docs/connect-blockchain/

Example integration reference -- https://web3auth.io/docs/connect-blockchain/tezos

**Comment by TheSohit** [comment]
is there any sample references for bitcoin specifically?

Thanks
